lib/fluent/plugin/in_http.rb

```diff
@@ -643,6 +643,10 @@ def include_cors_allow_origin
 
         !r.nil?
       end
+
+      def parse_query(query)
+        query.nil? ? {} : Hash[URI.decode_www_form(query, Encoding::ASCII_8BIT)]
```

Comment:
';' separator to be ignored. Is it ok here?
(I'm not sure difference WEBrick::HTTPUtils.parse_query and URI.decode_www_form)

Reply:
ref. https://rubydoc.info/stdlib/webrick/1.8.6/WEBrick%2FHTTPUtils:parse_query

Reply:
Oh, I didn't notice that.

If we want to keep that specification, it would be better not to replace it by force.

> `WEBrick::HTTPUtils` is just a util, so it might be harmless.

Reply:
Seems [puma](https://github.com/puma/puma) that Ruby web server also uses URI.decode_www_form.
https://github.com/puma/puma/blob/97c7d129a940c809fb379b3ecf314d39a18a332b/lib/puma/util.rb#L29

Reply:
Thanks! I see!